Project2.md

```diff
@@ -58,13 +54,68 @@ but I think it's a reasonable short-cut for a class project. The risk
 with this is that it could expose security vulnerabilities so you
 wouldn't want to really put it into production.
 
+Docker Compose
+--------------
+
+One other thing to introduce before we start coding is Docker Compose.
+Compose is a tool for defining the topology and running multi-container Docker applications. 
+With Compose, you use a Compose file to configure your application’s services.
+Recall how we start web, mysql containers one by one in the previous project. This
+approach works fine when we have only two containers and the configuration is simple, but
+is certainly not scalable. Using Docker Compose, we will be able to stand up our containers
+using a single command. Here is a tutorial about how Docker Compose works:
+
+First, install Docker Compose by doing
+
+	$ curl -L https://github.com/docker/compose/releases/download/1.8.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
```

Comment:
Do you need to install Compose separately or is it part of Docker for Mac/Windows?